tests/pad.test.ts

```diff
@@ -0,0 +1,17 @@
+import { expect, test } from "bun:test"
+import { fp } from "../src/footprinter"
+import { getTestFixture } from "./fixtures/get-test-fixture"
+
+test("pad footprint", async () => {
+  const { snapshotSoup } = await getTestFixture("pad")
+  const soup = fp().pad().w(2).h(1).circuitJson()
+  expect(soup).toMatchSnapshot()
```

Comment:
don't use `toMatchSnapshot`, you can use `toMatchInlineSnapshot` but in this case `snapshotSoup` is better